syntax = "proto3";
service Bookings {
  rpc GetBookingByUserID(UserID) returns (stream Date) {}
  rpc AddBookingByUserID(UserID) returns (Result){}
  rpc GetBookings(Empty) returns (BookingObject){}
}

message BookingObject{
  string userid= 1;
  repeated Date dates= 2;
}

message Date{
  string date = 1;
  repeated string movieId = 2;
}

message MovieID{
  string date = 1;
}

message UserID{
  string id=1;
}

message Empty {
}

message Result {
  string message= 1;
}